model/model.md

```diff
@@ -12,9 +12,9 @@ The relationships between the primary dataspace entities are defined as follows:
 
 Note that all relationships are multiplicities unless specified.
 
-- A `Dataspace Authority` manges one or more `Dataspaces`. This will include participant registration and may entail mandating business and/or requirements. For example, a
+- A `Dataspace Authority` manages one or more `Dataspaces`. This will include participant registration and may entails mandating business and/or requirements. For example, a
```

Comment:
```suggestion
- A `Dataspace Authority` manages one or more `Dataspaces`. This will include participant registration and may entail mandating business and/or requirements. For example, a
```
I think "entail" is correct in combination with the "may".